recipes/server.rb

```diff
@@ -20,6 +20,14 @@
 node.default['percona']['skip_passwords'] = false
 node.default['percona']['server']['bind_address'] = '0.0.0.0'
 
+# Loosen restrictions on type of functions users can create
+node.default['percona']['conf']['mysqld']['log_bin_trust_function_creators'] = '1'
+
+# utf8mb4 support
+node.default['percona']['conf']['mysqld']['innodb_large_prefix'] = 'true'
+node.default['percona']['conf']['mysqld']['innodb_file_format'] = 'barracuda'
+node.default['percona']['conf']['mysqld']['innodb_file_per_table'] = 'true'
```

Comment:
This is already set [here](https://github.com/osuosl-cookbooks/osl-mysql/blob/master/recipes/server.rb#L38).